spec/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source_spec.rb

```diff
@@ -59,6 +59,9 @@
     repo.create_branch("other_branch")
     # END: Setup local repo used for this spec
 
+    # Stub REPO_DIR for travis
+    stub_const("#{described_class}::REPO_DIR", Pathname.new(Dir.mktmpdir))
```

Comment:
Can we change this so the `repo_dir` `let` call is:

`let(:repo_dir) { Pathname.new(Dir.mktmpdir) }`

Then we can do the `stub_const` here as `stub_const("#{described_class}::REPO_DIR", repo_dir)`

I don't want to get into a situation where `repo_dir` can be called in the spec before we stub the constant and end up with the incorrect value cached.

Reply:
Fixed.  Good call!

Also, I forgot I had the `let(:repo_dir)` in there... oops...